doc/source/usage/tutorial/Parallelization/Direct_chunk_read.ipynb

```diff
@@ -0,0 +1,1126 @@
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<TimeitResult : 50.7 s ± 0 ns per loop (mean ± std. dev. of 1 run, 1 loop each)>"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%timeit -r1 -n1 -o -q\n",
+    "#Saving of an HDF5 file with many frames ...\n",
+    "with h5py.File(filename, \"w\") as h:\n",
+    "    ds = h.create_dataset(\"data\", shape=(nbframes,)+shape, chunks=(1,)+shape, dtype=dtype, **cmp) \n",
+    "    for i in range(nbframes):\n",
+    "        ds[i] = frame + i%500 #Each frame a a different (random) value to prevent caching effects"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "f82b2daf-77c2-4720-9848-6ca9c273707f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "File size 9.237 GB with a compression ratio of 7.410x\n",
+      "Write speed: 1450.535 MB/s of uncompressed data\n"
+     ]
+    }
+   ],
+   "source": [
+    "timing_write = _\n",
+    "size=os.stat(filename).st_size\n",
+    "print(f\"File size {size/(1024**3):.3f} GB with a compression ratio of {nbframes*numpy.prod(shape)*dtype.itemsize/size:.3f}x\")\n",
+    "print(f\"Write speed: {nbframes*numpy.prod(shape)*dtype.itemsize/(1e6*timing_write.best):.3f} MB/s of uncompressed data\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dc086333-96da-47eb-8477-9a0a5f60bc20",
+   "metadata": {},
+   "source": [
+    "Reading all data from HDF5 file is as slow if not slower than the writing. This is mostly due to the decompression and to the many memory allocation performed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "a643e63d-1709-45b1-aaad-a542792b5a62",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<TimeitResult : 28.4 s ± 0 ns per loop (mean ± std. dev. of 1 run, 1 loop each)>"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%timeit -r1 -n1 -o -q\n",
+    "#Reading all frames and decompressing them\n",
+    "buffer = numpy.zeros(shape, dtype=dtype)\n",
+    "with h5py.File(filename, \"r\") as h:\n",
+    "    ds = h[\"data\"]\n",
+    "    for i in range(nbframes):\n",
+    "        ds.read_direct(buffer, numpy.s_[i,:,:], numpy.s_[:,:])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "id": "847240c6-f9df-4626-b1a3-2197f535897c",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Read speed: 2585.264 MB/s of uncompressed data.\n"
+     ]
+    }
+   ],
+   "source": [
+    "timing_read1 = _\n",
+    "print(f\"Read speed: {nbframes*numpy.prod(shape)*dtype.itemsize/(1e6*timing_read1.best):.3f} MB/s of uncompressed data.\")"
```

Comment:
It would be nice to have a `fps` here as well to compare with following figures given as `fps`.